src/k2/cmd/tpcc/schema.h

```diff
@@ -39,7 +39,7 @@ static const String tpccCollectionName = "TPCC";
 #define CHECK_READ_STATUS(read_result) \
     do { \
         if (!((read_result).status.is2xxOK())) { \
-            K2LOG_D(log::tpcc, "TPC-C failed to read rows: {}", (read_result).status); \
+            K2LOG_E(log::tpcc, "TPC-C failed to read rows: {}", (read_result).status); \
```

Comment:
We should keep these and the three other log lines in the helper functions as debug because they can be normal expected failures during the benchmark (e.g. conflict abort)

Reply:
I see. Reverted back.